udagram-api-user/src/controllers/v0/users/routes/auth.router.ts

```diff
@@ -8,7 +8,6 @@ import * as jwt from 'jsonwebtoken';
 import {NextFunction} from 'connect';
 
 import * as EmailValidator from 'email-validator';
-import {config} from 'bluebird';
```

Comment:
Any reason for this? Was it imported by mistake?

Reply:
Unused import